public/plugin.ts

```diff
@@ -76,6 +77,8 @@ export class FlowFrameworkDashboardsPlugin
     setDataSourceManagementPlugin(plugins.dataSourceManagement);
     const enabled = !!plugins.dataSource;
     setDataSourceEnabled({ enabled });
+    console.log('datasourceManagement', plugins.dataSourceManagement);
+    console.log('data source', plugins.dataSource);
```

Comment:
Kama, I see few print statements in here. Could you remove them?

Reply:
Yes will remove this